documentation/docs/steps/transportRequestUploadRFC.md

```diff
@@ -0,0 +1,101 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+* You have enabled RFC on the ABAP system.
+* You have an user account on the ABAP system and the roles required for uploading via RFC.
+* You have created a transport request on the ABAP system, which is the target container of the upload.
+* You have installed an RFC library based Connector Client. See [RFC Client](#[RFC Client).
+
+## RFC Client
+
+Create a Docker image as described in the Git repository [devops-docker-images/node-rfc](https://github.com/rodibrin/devops-docker-images/tree/master/node-rfc).
+Put the image in its own Docker register in [Docker Hub](https://hub.docker.com/).
+Make the image known to the pipeline in the configuration.
+
+```yaml
+steps:
+  transportRequestUploadRFC:
+    dockerImage: 'my/rfc-client'
+```
+
+The RFC Client connects to the ABAP system using the [SAP NetWeaver RFC SDK](https://support.sap.com/en/product/connectors/nwrfcsdk.html).
+See the documentation of the [classical SAP connectivity technology RFC](https://help.sap.com/viewer/753088fc00704d0a80e7fbd6803c8adb/1709%20000/en-US/4888068ad9134076e10000000a42189d.html) for detailed information.
+
+## Specifying the Transport Request
+
+The target of the upload is a transport request, identified by an identifier (ID).
+The step `transportRequestUploadRFC` allows you to set the ID by parameter.
+Alternatively, you can pass the ID through the Common Pipeline Environment.
```

Comment:
Why is "Common Pipeline Environment" written in capital letters? 
I suggest: ".. a common pipeline environment."